common/ferrostar/src/routing_adapters/error.rs

```diff
@@ -57,6 +57,8 @@ pub enum ParsingError {
     // TODO: Unable to find route and other common errors
     #[cfg_attr(feature = "std", error("Failed to parse route response: {error}."))]
     ParseError { error: String },
+    #[cfg_attr(feature = "std", error("Failed to parse annotations: {error}."))]
+    Annotations { error: String },
```

Comment:
Probably worth a look now that we have two variants revisiting `ParseError`. I think I wrote that and I now think the name may be too vague. Can we make it more specific?